web_console/web-console-overview.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="web-console-overview"]
+= Web Console Overview
+include::_attributes/common-attributes.adoc[]
+:context: web-console-overview
+
+toc::[]
+
+The Red Hat {product-title} web console provides a graphical user interface to visualize your project data and perform administrative, management, and troubleshooting tasks. The web console runs as pods on the control plane nodes in the openshift-console project. It is managed by an operator pod. Both *Administrator* and *Developer* perspectives are supported.
+
+Both *Administrator* and *Developer* perspectives enable you to create quick start tutorials for the OpenShift Container Platform. A quick start is a guided tutorial with user tasks and is useful for getting oriented with an application, operator, or other product offering.
```

Comment:
When you say "create quick start," do you mean actually creating a new quick start or just running through one? We might rephrase if the latter.

Reply:
It looks like creating a new one based on what we have in the docs https://docs.openshift.com/container-platform/4.10/web_console/creating-quick-start-tutorials.html